addons/web/static/tests/helpers/mock_server.js

```diff
@@ -373,17 +401,19 @@ var MockServer = Class.extend({
      * @private
      * @param {string} model a model name
      * @param {any[]} domain
+     * @param {Object} param2
+     * @param {boolean} [param2.active_test=true]
      * @returns {Object[]} a list of records
      */
-    _getRecords: function (model, domain) {
+    _getRecords: function (model, domain, { active_test = true } = {}) {
```

Comment:
where do you need this unset?

Reply:
For example getting info from odoo bot and public partner which are archived.